exercises/exercise_000_sudoku_solver_initial_state/IDE_setup.md

```diff
@@ -0,0 +1,20 @@
+# Setting up IntelliJ or Visual Code Studio for Scala development
+
+## Setting up IntelliJ for Scala development
+
+- If you haven't already installed the IntelliJ IDE, follow the [Getting started](https://www.jetbrains.com/help/idea/installation-guide.html) instructions on the JetBrains website. The Community Edition should be sufficient for this workshop.
+
+- After installing the IDE, install the IntelliJ Scala plugin by following [these instructions](https://www.jetbrains.com/help/idea/discover-intellij-idea-for-scala.html#scala_plugin).
+
+## Setting up Visual Code Studio for Scala development
+
+- Download the installation binary for your system (Windows, Mac, Linux) [here](https://code.visualstudio.com/download).
+
+- Follow the set-up instructions [here](https://code.visualstudio.com/docs/setup/setup-overview).
+
+- Install the Metals plugin for Visual Code Studio by following the instructions [here](https://scalameta.org/metals/docs/editors/vscode/)
+
+
+## Other Scala source editors
+
+There are plenty of other options for Source Code editing. For example, [Scala] Metals supports Vim, Sublime Text and Emacs. For more information, have a look at the [Text Editors section](https://scalameta.org/metals/docs) in the Metals Documentation
```

Comment:
Why are you using square brackers here "[Scala] Metals"?

Reply:
Good question. Looking at the Metals doc, it says _"Scala language server with rich IDE features"_.

I'll change it to "Metals (Scala language server with rich IDE features)"